code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -720,103 +181,151 @@
 	icon_state = "setup_large_industrial"
 	desc = "It's a case, for building large electronics with. This one resembles some kind of industrial machinery."
 
-/obj/item/electronic_assembly/drone
+/********
+ * Mobs *
+ ********/
+/mob/living/integrated_drone
```

Comment:
is this fine just being a mob/living? when I had it as a simple mob I had to use some really bad define stuff to make it work so I reverted it to this.